src/ServiceControl/CustomChecks/Web/StatisticsResult.cs

```diff
@@ -0,0 +1,11 @@
+namespace ServiceControl.CustomChecks
+{
+    using System.Collections.Generic;
+
+    public class StatisticsResult
```

Comment:
This pattern might come up a lot. I wonder if this should be something like:

```csharp
class QueryResult<T>
{
  public IList<T> Results { get; set; }
  public int TotalResults { get; set; }
  public string ETag { get; set; }
}
```

We do already have something like this for scatter-gather.

We may want to hold off on doing this if we only have one instance at the moment.

Reply:
I refactored to use the existing QueryResult<T> class.